cmd/skipper/main.go

```diff
@@ -454,6 +450,7 @@ func main() {
 		log.Fatal(err)
 	} else {
 		log.SetLevel(logLevel)
+		fmt.Printf("set log level to %q\n", logLevel)
```

Comment:
why not log.Debugf() here ?

Reply:
fmt.Printf prints to stdout. Whichever log level we use here, it should print to stderr.

Reply:
May be it should be in front and use the standard log level (info?)?

Reply:
This is a big **OOPS!**. It was put there only for a making sure the log level was changed. Personal observation. And was forgotten there. I will remove it.

Reply:
And it was `Printf` because I could not see my change of log level applied at first. This is a plain-ol'-forgotten "debug-with-the-std::out" case.